src/app.js

```diff
@@ -663,37 +1342,120 @@ gpii.app.menu.getUserName = function (userToken) {
     return name;
 };
 
+
+/**
+*  Object representing options for a `Electron` `ContextMenu` item.
+ * @typedef {Object} ElectronMenuItem
```

Comment:
Very nice - now all we need is ... TYPESCRIPT... or some similar abomination : P